<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfEngagementStatistics</class>
 <widget class="QDialog" name="WkfEngagementStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engagement Statistics</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attacker:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="wkf::GroupingSelector" name="mAttacker">
     <property name="minimumSize">
      <size>
       <width>285</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Platform Type: WSF_PLATFORM</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="wkf::GroupingSelector" name="mTarget">
     <property name="minimumSize">
      <size>
       <width>285</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Platform Type: WSF_PLATFORM</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mEngagementsLabel">
     <property name="text">
      <string>Engagements:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="UtQtTablePlotWidget" name="mTablePlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wkf::GroupingSelector</class>
   <extends>QToolButton</extends>
   <header>selectors/WkfGroupingSelector.hpp</header>
  </customwidget>
  <customwidget>
   <class>UtQtTablePlotWidget</class>
   <extends>QWidget</extends>
   <header>UtQtTablePlotWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
